src/scss/components/_threeImages.scss

```diff
@@ -0,0 +1,45 @@
+.images{
+    margin: 60px 0;
+
+    &__item{
+        width: 80%;
```

Comment:
If you use the right grid classes in the template, you don't need to set this `width` here